heroku/cli
# refactor(orgs-v5): move teams to CLI & upgrade to oclif

### Description

This converts `teams` to oclif and updates its associated test.

### Testing

1) Run `./bin/run teams` with this branch pulled down
2) Verify the command succeeds and you see your list of teams
3) Compare the results with prod cli command and make sure things look 👍
